change-notes/1.20/analysis-python.md

```diff
@@ -24,6 +24,7 @@ Removes false positives seen when using Python 3.6, but not when using earlier v
  | **Query**                  | **Expected impact**    | **Change**                                                       |
 |----------------------------|------------------------|------------------------------------------------------------------|
 | Comparison using is when operands support \_\_eq\_\_ (`py/comparison-using-is`) | Fewer false positive results | Results where one of the objects being compared is an enum member are no longer reported. |
+| Modification of parameter with default (`py/modification-of-default-value`) | More true positive results | Instances where the mutable default value is mutated inside other functions are now reported correctly. |
```

Comment:
If we simply didn't report them before, consider changing to: "...functions are now also reported." Otherwise it implies that we used to report them but in the wrong way.

Reply:
Done. Thanks!